salt/elife-dashboard/init.sls

```diff
@@ -190,14 +196,15 @@ app-done:
     cmd.run: 
         - name: echo "app is done installing"
         - require:
-            - cmd: load-db-schema
-            - cmd: configure-{{ app.name }}
-            - cmd: npm-install
+            - load-db-schema
+            - configure-{{ app.name }}
+            - npm-install
 
 #
 # process queue
 #
 
+# TODO: this is an upstart service. how does it relate to daemons.conf in processes.sls
```

Comment:
have I forgotten to migrate something?

Reply:
This is the equivalent of `salt/elife-dashboard/config/lib-systemd-system-elife-dashboard-process-queue-daemon@.service` so now it is present.

Reply:
`daemon` in the original formula is a meaningless name, it really is a queue worker processing incoming SQS messages.

Reply:
I should remove that comment, it was more of a note-to-self. I ported this stuff processes stuff to systemd ages ago and was going to do a quick refresher